<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>291</height>
   </rect>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>273</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Today is:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="yearNumber">
      <property name="numDigits">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="monthNumber">
      <property name="numDigits">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="dayNumber">
      <property name="numDigits">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>201</width>
     <height>152</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>卡号：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="cardIdEdit">
      <property name="text">
       <string>00ff7f9d</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="nameEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="classLabel">
      <property name="text">
       <string>班级：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="classesEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="dormitoryLabel">
      <property name="text">
       <string>寝室楼栋：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="dormitoryEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="studentIDLabel">
      <property name="text">
       <string>学号：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="studentidEdit"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="stateEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>挂失状态：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font-family:heiti;
	font: 75 10pt &quot;黑体&quot;;
height:20px;
width:60px;
color:white;
background-color:rgb(1, 70, 39);
border-radius:8px;
}</string>
   </property>
   <property name="text">
    <string>确认进入</string>
   </property>
  </widget>
  <widget class="RotateNumberLabel" name="hourLabel">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>10</y>
     <width>48</width>
     <height>45</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>48</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="RotateNumberLabel" name="minuteLabel">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>10</y>
     <width>48</width>
     <height>45</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>48</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QLCDNumber" name="secondNumber">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>37</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="numDigits">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>请输入寝室楼栋号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ifdomitoryEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="jieguaBtn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font-family:heiti;
	font: 75 10pt &quot;黑体&quot;;
height:20px;
width:60px;
color:white;
background-color:rgb(1, 70, 39);
border-radius:8px;
}</string>
   </property>
   <property name="text">
    <string>解除挂失</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RotateNumberLabel</class>
   <extends>QLabel</extends>
   <header>rotatenumberlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
